lib/cpp/src/thrift/server/TThreadPoolServer.h

```diff
@@ -105,19 +105,16 @@ class TThreadPoolServer : public TServer {
       timeout_(0),
       taskExpiration_(0) {}
 
-  virtual ~TThreadPoolServer();
+  virtual ~TThreadPoolServer() {}
```

Comment:
I did this so that it would be the same as other classes in the project, as TSimpleServer and TThreadedServer do this.  If I change this I would change all three servers to be the same.

Personally, I don't like having implementations in headers at all even if they are trivial, because when you need to change the implementation (and not the signature of the call) you end up rebuilding too much.